samples/angular-todo/src/app/features/journey/webAuthn/webAuthn.component.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * angular-todo-prototype
+ *
+ * webAuthn.component.ts
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import {
+  FRAuth,
+  FRLoginFailure,
+  FRLoginSuccess,
+  FRStep,
+  TokenManager,
+  UserManager,
+} from '@forgerock/javascript-sdk';
+import FRWebAuthn from 'packages/javascript-sdk/src/fr-webauthn';
```

Comment:
can we use the above import?

Reply:
@ryanbas21 got this message "Projects cannot be imported by a relative or absolute path, and must begin with a npm scope"

Reply:
thats exactly what the above import is doing

Reply:
done

Reply:
done